### DISCUSSION TITLE
How to fix a possible permissions issue?

### DISCUSSION THREAD

**Discussion by dustinatx** [leokhoa/laragon#471] [Q&A]
I'm using Laragon 6 on Windows 11. I'm encountering problems with some plugins on a few WordPress installations. These seem to be related to permissions...

### WP Migrate

When pulling from production to local, I'm shown an error, "wp-content/mu-plugins is not writable at the destination."

wp-content/mu-plugins doesn't exist on the local installation. If I manually create this folder, the error goes away.

It seems that WordPress running on Laragon can't create this folder on its own.

### WP Rocket

Clearing the cache for WP Rocket doesn't work. Nothing happens. None of the files get deleted or re-created.

If I manually delete the cache folder, WP Rocket will build new files.

It seems that WordPress running on Laragon can't delete the cache folder on it's own.

Is this a permissions issue? I only have one account on this PC. It should have all permissions, as far as I know. I've tried running Laragon as an administrator, but this problem still happens.

Any ideas on how to solve this?

**Comment by Pen-y-Fan** [comment]
See my blog on permissions: [How to set up Laragon on a new Windows computer (part 2) - permissions](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-2/), it may help to apply permissions.

**Comment by dustinatx** [comment]
@Pen-y-Fan Thanks for your reply. I enabled the permissions from your post. Unfortunately, that had no impact on these issues.

On a side note, how can I get back to those Windows Defender screens? When I first ran Laragon, before reading your post, I closed those screens without allowing access for mysqld and httpd.

**Comment by dustinatx** [comment]
@Pen-y-Fan Disregard the Windows Defender question. I opened Windows Defender and deleted the existing rules for those applications. The next time I started services in Laragon I got the screens that let me allow access.
